src/tseda/datastore.py

```diff
@@ -71,31 +70,31 @@ class IndividualsTable(Viewer):
         default=100,
         doc="Number of rows per page to display",
     )
-    toggle = param.Integer(
-        default=None, bounds=(0, None), doc="Toggle sample set by index"
+    sample_select = pn.widgets.MultiChoice(
+        name="Select sample sets",
+        description="Select samples based on the sample set ID.",
+        options=[],
+        value=[0, 1],
     )
     population_from = param.Integer(
```

Comment:
should we add a warning box if you enter an invalid population id? such as the one on the tree page?

Reply:
Yes, added as feature request #56